### PR TITLE
fixes issue #18 add -l option for list of x and y arrays to plot points

### PR DESCRIPTION
**Pull request by LoneOne123**
everything said in title ;)

### CODE REVIEW

File: lib/plotutil.py [hunk 1]
```diff
@@ -58,3 +58,40 @@ def plot(func, xstart, xend, step, color_name, xlabel, ylabel, gui):
...
+			if not color_name in colors:
+				print color_name,": Color not found."
+				color_name = 'blue'
+		xvals = list(map(float, arrays[1:arrays.find(']')].split(','))) #extract numbers from X-array
+		yvals = list(map(float,arrays[arrays.find(']')+3:len(arrays)-1].split(','))) #extract numbers from Y-array
+		if len(xvals) == len(yvals):
+			plt.plot(xvals, yvals, color=color_name, linewidth=2.0)
+		else:
+			print('Error: You need same number of X and Y values')
+		plt.xlabel(xlabel)
```
Comment (Aniq55) [inline lib/plotutil.py]: Put this within the len check if block

File: lib/plotutil.py [hunk 2]
```diff
@@ -58,3 +58,40 @@ def plot(func, xstart, xend, step, color_name, xlabel, ylabel, gui):
...
+		plt.ylabel(ylabel)
```
Comment (Aniq55) [inline lib/plotutil.py]: Put this within the len check if block

File: lib/plotutil.py (hunk 1 above)
Comment (Aniq55) [reply lib/plotutil.py]: @LoneOne123

File: lib/plotutil.py (hunk 2 above)
Comment (LoneOne123) [reply lib/plotutil.py]: @Aniq55 ok changed it. is it done with the commit or need i do something with the pr here that the commit is linked to this pr? some git related stuff is still a bit confusing for me :D

**Comment by djmgit** [comment]
@LoneOne123 Looks good, although next time please try to squash your commits into one. Squashing means simply merging your multiple commits (2 in this case) into one. Ideally there should be one commit per PR, this helps in maintaining a clean history. Once @Aniq55 reviews this PR, it will be ready to go :+1:

**Comment by Aniq55** [comment]
@djmgit it's cool. Squash and Merge? we can do that if he hasn't.

**Comment by LoneOne123** [comment]
that would be this command line right?
git rebase -i HEAD~2

**Comment by Aniq55** [comment]
@LoneOne123 yeah!

**Comment by LoneOne123** [comment]
hmm think i dont get it workin i will read a bit more and hopefully i get it then into 1 commit

**Pull request closed without merging**